MyOnboarding/test/MyOnboardingApp.Services.Tests/Criteria/DateCheckingCriteria/LastUpdateDateBiggerThanCreationDateCriterionTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Linq;
+using MyOnboardingApp.Contracts.Models;
+using MyOnboardingApp.Contracts.Validation;
+using MyOnboardingApp.Services.Criteria.DateCheckingCriteria;
+using MyOnboardingApp.TestUtils.Extensions;
+using MyOnboardingApp.TestUtils.Factories;
+using NUnit.Framework;
+
+namespace MyOnboardingApp.Services.Tests.Criteria.DateCheckingCriteria
+{
+    [TestFixture]
+    public class LastUpdateDateBiggerThanCreationDateCriterionTests
+    {
+        private IValidationCriterion<TodoListItem> _dateCriterion;
+
+
+        [SetUp]
+        public void SetUp()
+            => _dateCriterion = new LastUpdateDateBiggerThanCreationDateCriterion();
+
+
+        [Test]
+        public void Validate_ValidItemSpecified_ReturnsEmptyCollection()
+        {
+            var testItem = ItemVariantsFactory.CreateItemVariants(
+                id: new Guid("00000000-0000-0000-0000-000000000001"),
+                text: "test item",
+                creationTime: new DateTime(2015, 01, 01),
+                lastUpdateTime: new DateTime(2015, 02, 02)
+            ).Item;
+            var expectedErrorLocation = new string[] { };
+
+            var result = _dateCriterion
+                .GetErrorLocationsUsingFakeErrorFactory(testItem)
+                .ToArray();
+
+            Assert.That(result, Is.EquivalentTo(expectedErrorLocation));
+        }
+
+
+        [Test]
+        public void Validate_CreationTimeBiggerThanLastUpdateTimeSpecified_ReturnsCollectionWithError()
+        {
+            var testItem = ItemVariantsFactory.CreateItemVariants(
```

Comment:
```suggestion
            var (testItem, _, _) = ItemVariantsFactory.CreateItemVariants(
```
Should be perfectly valid as well and seem a bit clearer to me (let's discuss in person).